tests/cupy_tests/functional_tests/test_vectorize.py

```diff
@@ -362,8 +367,12 @@ def func_if(x):
             return y
 
         f = xp.vectorize(func_if)
-        x = xp.array([1, 2, 3, 4, 5])
-        return f(x)
+        x = xp.array([1, 2, 3, 4, 5], dtype=xp.int64)
+        out = f(x)
+        # On Windows, NumPy's default int type is int32, not int64
+        if sys.platform.startswith('win32') and out.dtype == xp.int32:
+            out = out.astype(xp.int64)
+        return out
```

Comment:
How about fixing the type of `y` in the target function instead? This test is for if-statement, so the result dtype is not so important.
```py
        def func_if(x):
            y = numpy.int64(0)
            if x % 2 == 0:
                y = x
            return y
```

Reply:
`y = numpy.int64(0)` works in the function to be jitted?! Wow...OK will give it a try.

Reply:
Just now I'm considering another fix.
https://github.com/cupy/cupy/blob/v9.0.0b3/cupyx/jit/_typerules.py#L121
Could you try changing this line to `return _types.Scalar(int)`?

Reply:
> Could you try changing this line to `return _types.Scalar(int)`?

Thanks, @asi1024! I reverted all test changes (537a5a8) and then did this (d55549c). At least all tests pass on Linux, so let's see if Windows complains 🙂